src/clj/rems/workflow/dynamic.clj

```diff
@@ -534,8 +599,23 @@
     :member {:userid "member"}}
    {:type ::invite-member
     :actor actor
-    :member {:name "Name"
-             :email "email@address.org"}}])
+    :member {:name "name"
+             :email "email@address.org"}}
+   (let [members (->> application-state
+                      :members
+                      (remove (comp #{(:applicantuserid application-state)} :userid)))]
+     (when (seq members)
+       {:type ::remove-member
+        :actor actor
+        :member (first members)
+        :comment "comment"}))
```

Comment:
Why not when-let? And does it make sense to allow removing the last member of the application, which might be the applicant?

Reply:
Checking for `members` passes with empty collection whereas `(seq members)` means non-empty collection?

I'm ok with the applicant removing himself/herself for example if he or she wants to get out of the team. There should be a way for the handler to promote another member to main applicant. Maybe it makes sense to offer this to the applicant as well.